draft-ietf-rats-corim.md

```diff
@@ -1023,6 +1024,9 @@ A cryptographic key can be one of the following formats:
   certificates MUST be concatenated in order so that each directly certifies
   the one preceding.
 
+* `tagged-pkix-asn1der-key-type`: ASN.1 DER encoded X.509 public key certificate.
```

Comment:
We want to say this DER-encoded cert is `bstr`-wrapped.

Also, it looks like `cddl/crypto-key-type-choice.cddl` needs to be informed about this new type.

Reply:
Done.